README.md

```diff
@@ -221,6 +221,17 @@ Run the integration test suite inside a containerized instance of Firefox. You m
 
 Start a linux VM container with VNC available over `vnc://localhost:5900` with password `secret`. Right click on the desktop and select `Applications > Shell > Bash` and enter `tox -c tests/integration/` to run the integration tests and watch them run in a Firefox instance you can watch and interact with.
 
+## Accessing Remote Settings locally
+
+In development you may wish to approve or reject changes to experiments as if they were on Remote Settings. You can do so here: `http://localhost:8888/v1/admin/`
```

Comment:
Will you add this to the list of important links? 😁 (https://github.com/mozilla/experimenter#important-links)

Reply:
@LZoog sure, can you tell me the address for prod/stage?

Reply:
Beat me to it!

prod: https://settings-writer.prod.mozaws.net/v1/admin/#/
stage: https://settings-writer.stage.mozaws.net/v1/admin/#/

Reply:
Done!